85 clean track assets

luigiberducci opened the pull request:
Closes #85.

- Updated generator legacy scan to work with current version of the simulator. 
- Replace legacy scan file using 3 maps Spielberg, Monza, Austin.
- Remove old maps (without centerline/raceline) and update tests accordingly.

hzheng40 commented on .gitignore:
```diff
@@ -170,5 +170,5 @@ cython_debug/
 *.zip
 
-/gym/f110_gym/maps/*
+/maps/*
```
should be **/maps/*?

luigiberducci replied:
corrected

hzheng40 commented on maps/Spielberg/Spielberg_DonkeySim_waypoints.txt:
These should not be included? Probably because the gitignore wasn't working

hzheng40 commented on maps/Spielberg/Spielberg_centerline.csv:
Shouldn't be included.

hzheng40 commented on maps/Spielberg/Spielberg_map.png:
Shouldn't be included.

hzheng40 commented on maps/Spielberg/Spielberg_map.yaml:
Shouldn't be included.

hzheng40 commented on maps/Spielberg/Spielberg_raceline.csv:
Shouldn't be included.

hzheng40 reviewed (changes requested):
Small issue with gitignore

luigiberducci commented:
@hzheng40 I have corrected the gitignore and deleted the old maps Berlin/Skirk from maps.

Also, Spielberg has been removed from maps/. Now at the first run, the user is gonna download the necessary map.
Do we want to include 1 sample map or not?

Apart from that, if it is fine for you, we can close this.

hzheng40 commented:
I think downloading is fine.
